oci/spec.go

```diff
@@ -220,3 +223,25 @@ func populateDefaultDarwinSpec(s *Spec) error {
 	}
 	return nil
 }
+
+// FromDescriptorProto converts containerds protobuf [types.Descriptor]
+// to the OCI image specs [ocispec.Descriptor].
+func FromDescriptorProto(d *types.Descriptor) ocispec.Descriptor {
```

Comment:
Maybe we can change the pattern to `Kind[From|To]Proto` (`DescriptorFromProto`) which is more consistent with some existing functions and with cases where they're put in a package (e.g., `mount.FromProto`)

https://github.com/containerd/containerd/blob/9b4ed8acc2a04a3f8df140e79052d18b750d757e/services/containers/helpers.go#L38

https://github.com/containerd/containerd/blob/659a41fd8509baee82fdc217be37e4642c62f95f/pkg/transfer/image/imagestore.go#L390

Reply:
Also noticed `Platform` has a few duplication 😁 

https://github.com/containerd/containerd/blob/beabd3d5d1bc610bcddbcb7d94f4d7f0b3f41fd2/services/introspection/local.go#L192-L199

https://github.com/containerd/containerd/blob/659a41fd8509baee82fdc217be37e4642c62f95f/pkg/transfer/image/imagestore.go#L390

https://github.com/containerd/containerd/blob/ded713010c745c6e0010d7e0cec4dccd995e1a42/pkg/transfer/archive/exporter.go#L160

https://github.com/containerd/containerd/blob/5885db62c856e703f11f2195c54efbec5d00a788/cmd/ctr/commands/plugins/plugins.go#L152

Reply:
> Maybe we can change the pattern to Kind[From|To]Proto (DescriptorFromProto) 

Works for me!

Reply:
> Also noticed Platform has a few duplication 

The journey continues 😅

Reply:
@djdongjin Changed, and added a conversion from []Platform to []*types.Platform. I went with a slice as the only time we ever convert is for multiple at a time.